docs/getting-started/built-ins.md

```diff
@@ -0,0 +1,134 @@
+---
+sidebar_position: 8
+---
+
+# Built-ins
+
+## Functions
+
+The following functions come with `scrypt-ts` and are available globally.
+
+### Assert
+
+- `assert(condition: boolean, msg?: string): asserts condition`
+
+### Math
+
+- `abs(a: bigint): bigint  `
+- `min(a: bigint, b: bigint): bigint`
+- `max(a: bigint, b: bigint): bigint`
+- `within(x: bigint, min: bigint, max: bigint): boolean`
+
+### Hashing
+
+- `ripemd160(a: ByteString): Ripemd160`
+- `sha1(a: ByteString): Sha1`
+- `sha256(a: ByteString): Sha256`
+- `hash160(a: ByteString): Ripemd160`, actually returns `ripemd160(sha256(a))`
+- `hash256(a: ByteString): Sha256`, actually returns `sha256(sha256(a))`
+
+### Bytes Operation
+
+- `int2ByteString(n: bigint, size?: bigint): ByteString`. If not passing `size`,  the number `n` is converted to a `ByteString` with as few bytes as possible. Otherwise, converts the number `n` to a `ByteString` of the specified size, including the sign bit, fails if the number cannot be accommodated.
+- `byteString2Int(a: ByteString): bigint`
+- `len(a: ByteString): number`, returns the byte length of `a`
+- `reverseByteString(b: ByteString, size: number): ByteString`
+
+### Bitshift
+
+- `lshift(x: bigint, n: bigint): bigint`
+- `rshift(x: bigint, n: bigint): bigint`
+- `pow2(n: bigint): bigint`
+- `simplePow(x: bigint, y: bigint)`
+
+### Exit
+
+- `exit(status: boolean): void`. Call this function will terminate contract execution. If `status` is true then the contract succeeds; otherwise, it fails.
+
+## Library
+
+A library is the same with a contract, except it does not contain any public function. It is only intended to be imported by a contract or other libraries. It thus cannot be independently deployed and called. It is frequently used to group related constants and static functions.
```

Comment:
Delete, now https://scrypt.io/scrypt-ts/getting-started/how-to-publish-a-contract